public/index.html

```diff
@@ -14,7 +14,7 @@
   <body>
     <noscript>
       <strong>
-        We're sorry but BC Business Registry doesn't work properly
+        We're sorry but BC Registries doesn't work properly
```

Comment:
If this is about the specific Business Registry app, it should probably be "...but the Business Registry" doesn't work probably..." unless it pertains to all of BC Registries

Reply:
This is only for Filings UI. Changed to "the Business Registry app".

Reply:
hi @severinbeauvais - please hold off on making more updates tonight. I didn't know you are still online and I wanted to review my flagged ones with Scott in the morning.

Reply:
OK.

Reply:
@severinbeauvais Change to "JavaScript must be enabled for this application. Please enable it to continue."

Reply:
I like that.

Done!